imgr/imgrpkg/api.go

```diff
@@ -3,6 +3,9 @@
 
 // Package imgrpkg implements the server side Inode Management for ProxyFS volumes.
 // The package supports to communications protocols. The balance of the package
+
+====>>> The line "The package supports to communications protocols." needs to be reworded
```

Comment:
Good catch!

Reply:
BTW - this line wasn't commented... so it blew up the Travis-CI run...

Reply:
Yes - I know.    I wanted it to stick out and I could not find a way in github to just comment an existing file

I plan to delete this branch as soon as you are finished looking at ti.

Reply:
I reworded that part to read:

// Package imgrpkg implements the server side Inode Management for ProxyFS volumes.
// While the package provides a small set of Go-callable APIs, the bulk of its
// functionality is accessed via package retryrpc-exposed RPCs. While these RPCs
// reference active volumes known to an imgrpkg instance, a RESTful API is provided
// to specify those active volumes.
//
// Note that func's listed under type RetryRPCServerStruct are the RPCs issued by
// the client side via package retryrpc connections (and, thus, not intended to be
// called directly).

And, later, since the section was then redundant:

// The RESTful API is provided by an embedded HTTP Server
// (at URL http://<PrivateIPAddr>:<HTTPServerPort>) responsing to the following:

Reply:
Looks good to me